src/main/resources/application.properties

```diff
@@ -10,3 +10,7 @@ spring.datasource.password=${DB_PASS}
 
 #Do nothing
 spring.jpa.hibernate.ddl-auto=none
+
+#Mailgun email sender properties:
+mailgun.domain: sandbox08786ed216874ad697286dcf97357a43.mailgun.org
+mailgun.api.key: 41a98182603ad234d941c01860d8e426-c8c889c9-270b8e5a
```

Comment:
Put these as environment variables. The repo is public and you probably want to generate a new one now.

Reply:
ahh good point, removing

Reply:
Rotated key, and moved those variables into environment variables

Reply:
Checks are passed @Richard-outreach up to you if you wanna push it through. Pinging for notice.